proto/hts/participant/service.proto

```diff
@@ -52,28 +52,44 @@ message IsEventAvailableRequest {
 }
 
 service ParticipantService {
+    // Check whether event's start date is before specified date
```

Comment:
So we now need to comment on each method?

Reply:
Heard Five talk about it